Dockerfile

```diff
@@ -29,37 +29,26 @@ RUN apt-get update \
     && apt-get -y install \
         wget \
         python2 \
-        openjdk-8-jdk-headless \
+        openjdk-17-jdk-headless \
         patch \
         netcat \
         curl \
     && cd / \
     && export MAVEN_OPTS="-Xms2g -Xmx2g" \
-    && export JAVA_HOME="/usr/lib/jvm/java-8-openjdk-amd64" \
+    && export JAVA_HOME="/usr/lib/jvm/java-11-openjdk-amd64" \
     && tar -xzvf /apache-atlas-3.0.0-SNAPSHOT-server.tar.gz -C /opt \
```

Comment:
We have done 17 above and 11 here?

Reply:
Actually this was the state with what I did all the tests so far, fortunately it didn't break anything, still I have addressed this